vault/resource_mfa_totp_test.go

```diff
@@ -0,0 +1,45 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestMFATOTPBasic(t *testing.T) {
+	mfaTOTPPath := acctest.RandomWithPrefix("mfa-totp")
```

Comment:
Shadow declaration of `mfaTOTPPath()`.

It's preferable to use generic variable names in this scope.
```suggestion
	path := acctest.RandomWithPrefix("mfa-totp")
```

Reply:
Fixed in 5d506d7